### DISCUSSION TITLE
Betting Cap Option?

### DISCUSSION THREAD

**Discussion by johanvdmeer** [b-inary/wasm-postflop#7] [General]
Hello,

How do I limit the allowable bets and raises on a particular betting round, (capped betting)
e.g. If I want to be able to solve a Fixed Limit Hold'em game-format.

Regards,

Johannes

**Comment by b-inary** [comment]
I am afraid there is no way to do this at the moment. You can edit the game tree manually from the "Preview & Edit Tree" dialog, but it is impractical to remove all such nodes. I will consider implementing this feature if there are enough requests.

**Comment by b-inary** [comment]
Now FLHE supported. You can enter the "XcYr" format in the raise field. For example, if the fixed bet size is 20 and the maximum number of raises is 3, you could type "20c3r".

**Reply by johanvdmeer** [reply]
Thank you very much
